app/controllers/concerns/sufia/works_controller_behavior.rb

```diff
@@ -7,7 +7,6 @@ module WorksControllerBehavior
 
     included do
       self.show_presenter = Sufia::WorkShowPresenter
-      layout "sufia-one-column"
```

Comment:
@mjgiarlo any controller that includes on inherits from CurationConcerns::CollectionsControllerBehavior gets `with_themed_layout '1_column'` by default.  No need to override or set again.